src/custom/hooks/usePriceImpact/useFallbackPriceImpact.ts

```diff
@@ -1,19 +1,14 @@
 import { useEffect, useMemo, useState } from 'react'
 import { Percent } from '@uniswap/sdk-core'
 
-import { useDerivedSwapInfo } from 'state/swap/hooks'
-
 import useExactInSwap, { useCalculateQuote } from './useQuoteAndSwap'
-import { FallbackPriceImpactParams } from './types'
+import { FallbackPriceImpactParams, PriceImpactTrade } from './types'
 import { calculateFallbackPriceImpact } from 'utils/price'
-import TradeGp from 'state/swap/TradeGp'
 import { QuoteInformationObject } from 'state/price/reducer'
 import { QuoteError } from 'state/price/actions'
-import { useQuote } from 'state/price/hooks'
-import { useWeb3React } from '@web3-react/core'
 import { LegacyFeeQuoteParams } from '@cow/api/gnosisProtocol/legacy/types'
 
-type SwapParams = { abTrade?: TradeGp; sellToken?: string | null; buyToken?: string | null }
+type SwapParams = { abTrade?: PriceImpactTrade; sellToken?: string | null; buyToken?: string | null }
```

Comment:
Nice we get rid of those `Gp`. they come from the old name (Gnosis Protocol).

We should not have them